jujugui/static/gui/src/app/utils/statsd.js

```diff
@@ -0,0 +1,79 @@
+/*
+This file is part of the Juju GUI, which lets users view and manage Juju
```

Comment:
We have this now in the root of the project so aiui the short form can be used - see jujulib